src/test/java/test/TopologySpreadConstraintsTest.java

```diff
@@ -0,0 +1,78 @@
+package test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+import test.helm.Helm;
+import test.model.Product;
+
+import java.util.Map;
+
+import static test.jackson.JsonNodeAssert.assertThat;
+
+class TopologySpreadConstraintsTest {
+    private Helm helm;
+
+    @BeforeEach
+    void initHelm(TestInfo testInfo) {
+        helm = new Helm(testInfo);
+    }
+
+    @ParameterizedTest
```

Comment:
Isn't worth to add another test to verify `labelSelector`? 
Labels are the main detection mechanism to match the constrains in runtime.

Reply:
The selector test is now added, thanks for the reminder.